automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/Leonardo.scala

```diff
@@ -168,28 +155,4 @@ object Leonardo extends RestClient with LazyLogging {
       handleClusterResponse(patchRequest(url + path, clusterRequest))
     }
   }
-
-
-  sealed trait ApiVersion {
```

Comment:
I got rid of `ApiVersion` in the tests. All clients are using `v2` now -- `v1` has been deprecated for more than a year. An upcoming Leo PR will formally remove `v1` from our code.